api/nameguard/nameguard.py

```diff
@@ -92,6 +94,23 @@ def nested_get(dic, keys):
     return dic
 
 
+SIMPLE_NAME_REGEX = re.compile(r'^[a-z0-9]+\.eth$')
+
+
+def simple_name(name: str) -> bool:
+    return SIMPLE_NAME_REGEX.match(name) is not None
+
+
+def consolidated_report_from_simple_name(name: str) -> ConsolidatedNameGuardReport:
+    return ConsolidatedNameGuardReport(
+        name=name,
+        namehash=namehash_from_name(name),
+        normalization=Normalization.NORMALIZED,
+        rating=Rating.PASS,
+        risk_count=0,
+        highest_risk=None,
+    )
```

Comment:
@lightwalker-eth @Carbon225 Please review the regex and fields in the report - is it always true for any name matching the regex?
TODO: Now I see that Punycode check can fail for too long name.

Reply:
NameWrapper label is <=63